ci/scripts/connect-release-staging.sh

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env bash
+
+set -e -o pipefail
+
+# Usage:
+# ./connect-release.sh DESTINATION
+# @DESTINATION: required, destination directory (connect major version)
+
+# Validate params
+while [[ "$#" -gt 0 ]]; do case $1 in
+    9)
+        latest_version="9"
+        current_version=$(jq -r .version packages/connect/package.json)
+        shift
+        ;;
+    *)
+        echo "Invalid version parameter passed: $1
+Used only for version 9"
+        exit 1
+        shift
+        ;;
+    esac done
+
+echo "Uploading to s3://staging-connect.trezor.io/$latest_version/ and s3://staging-connect.trezor.io/$current_version/"
+
+# organize the files in one directory
+tmp_folder="tmp-connect-release"
```

Comment:
You could use "trap" to make sure that the temporary file is deleted when the scripts completes even if it does because of error like:

```bash
trap "rm -rf $tmp_folder" EXIT
```

Reply:
could be. I did not want to do many changes here. this file is based on already existing script and I don't have any power to test it properly before deploying this to production.